modules/gdscript/tests/scripts/runtime/errors/typed_array_assign_wrong_to_typed.out

```diff
@@ -0,0 +1,7 @@
+GDTEST_RUNTIME_ERROR
+>> ERROR
+>> on function: assign()
+>> core/variant/array.cpp
```

Comment:
This causes CI failures as on Windows the path is formatted differently (`core\variant\array.cpp`).

It's also going to fail whenever the line number changes in that file.

Reply:
Fixup: #78216